modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/ClassicEditor.js

```diff
@@ -100,7 +89,7 @@ const ClassicEditor = ({
 	}, [contents, getHTML, name]);
 
 	const onResize = debounce(() => {
-		setToolbarSet(getToolbarSet(initialToolbarSet));
+		setToolbarSet(initialToolbarSet);
```

Comment:
As the `toolbarSet` no longer changes with resize to `phone` and `tablet`, we no longer need the resize event listener.

Reply:
Sounds great, I can push that to the final solution, thanks for the quick review!

I think the new simple toolbar is if not simpler than phone at least as simple and I'm not sure the resizing method makes sense since it's not at all controlled by the user.